README.md

```diff
@@ -68,10 +68,11 @@ initialization, by passing `-e VAR=VALUE` to the Docker run command.
 
 Following environment variables influence PostgreSQL configuration file. They are all optional.
 
-|    Variable name              |    Description                                                          |    Default
-| :---------------------------- | ----------------------------------------------------------------------- | -------------------------------
-|  `POSTGRESQL_MAX_CONNECTIONS` | The maximum number of client connections allowed                        |  100
-|  `POSTGRESQL_SHARED_BUFFERS`  | Sets how much memory is dedicated to PostgreSQL to use for caching data |  32M
+|    Variable name              |    Description                                                                |    Default
+| :---------------------------- | ----------------------------------------------------------------------------- | -------------------------------
+|  `POSTGRESQL_MAX_CONNECTIONS` | The maximum number of client connections allowed                              |  100
```

Comment:
@jpechane you dont have to put additional spaces in lines where they are not required.
Pls check how its done in mysql image - https://raw.githubusercontent.com/openshift/mysql/master/README.md